modules/bibupload/lib/bibupload.py

```diff
@@ -2274,11 +2276,11 @@ def _process_document_moreinfos(more_infos, docname, version, docformat, mode):
 
 ### Update functions
 
-def update_bibrec_date(now, bibrec_id, insert_mode_p, pretend=False):
+def update_bibrec_date(now, bibrec_id, insert_mode_p, original_005=None, pretend=False):
     """Update the date of the record in bibrec table """
```

Comment:
If you add `original_005` not as the last optional argument, then you'll have to update all calls to it.  E.g. here:

```console
$ git grep -Hni update_bibrec_date
modules/bibupload/lib/bibupload_regression_tests.py:1545:        bibupload.update_bibrec_date(convert_datestruct_to_datetext(now), rec_id, False)
```